src/javascripts/app.js

```diff
@@ -6,9 +6,9 @@ import routes from './routes'
 import { AppContainer } from 'react-hot-loader'
 import { Router, browserHistory } from 'react-router'
 
-window.onload = function() {
-    document.getElementById("message-input").focus();
-};
+// window.onload = function() {
+//     document.getElementById("message-input").focus();
+// };
```

Comment:
go ahead and remove this -- you can always get code back with git history, so there's no need to keep these things around